src/lua/utils.c

```diff
@@ -554,17 +576,13 @@ luaT_tolstring(lua_State *L, int idx, size_t *len)
 	return lua_tolstring(L, -1, len);
 }
 
-/* Based on ffi_meta___call() from luajit/src/lib_ffi.c. */
```

Comment:
Typo: According to our [guidelines](https://www.tarantool.io/en/doc/latest/dev_guide/c_style_guide/#chapter-8-commenting) dots are required in comments. Same for the comment above.

Reply:
Fixed.